macros/macros.go

```diff
@@ -11,6 +11,7 @@ type EndpointTemplateParams struct {
 	PublisherID string
 	ZoneID      string
 	SourceId    string
+	AccountID   string
```

Comment:
Thank you for adding this. It makes sense for your use case.

However, it looks like this accidentally broke the `validateAdapterEndpoint` method in config.go. On line 266 there's a call to `macros.ResolveMacros` which includes hardcoded example data. Please add a dummy account id to the hardcoded `EndpointTemplateParams` data alongside `dummyHost` and `dummyPublisherID`.